### PR TITLE
Add postgresql evr_t type extension

### PR DESCRIPTION
**Pull request by ianballou**
Adds an RPM for the `evr_t` type PostgreSQL extension that will be used in Katello applicability.

### CODE REVIEW

File: comps/comps-katello-pulpcore-el7.xml [hunk 1]
```diff
@@ -97,7 +97,8 @@
       <packagereq type="default">python3-urllib3</packagereq>
       <packagereq type="default">python3-whitenoise</packagereq>
       <packagereq type="default">python3-yarl</packagereq>
-      <packagereq type="default">python3-zipp</packagereq>
+	    <packagereq type="default">python3-zipp</packagereq>
```
Comment (ekohl) [inline comps/comps-katello-pulpcore-el7.xml]: Looks like indenting went wrong here

File: mock/el7-scl.cfg [hunk 2]
```diff
@@ -1,7 +1,7 @@
 config_opts['root'] = 'foreman-scl-7-x86_64'
 config_opts['target_arch'] = 'x86_64'
 config_opts['legal_host_arches'] = ('x86_64',)
-config_opts['chroot_setup_cmd'] = 'install @buildsys-build scl-utils-build tfm-build yum vim'
+config_opts['chroot_setup_cmd'] = 'install @buildsys-build scl-utils-build yum vim rh-postgresql12-scldevel'
```
Comment (ekohl) [inline mock/el7-scl.cfg]: Any reason to drop `tfm-build`?

File: comps/comps-katello-pulpcore-el7.xml (hunk 1 above)
Comment (ianballou) [reply comps/comps-katello-pulpcore-el7.xml]: Interesting, I only see it in the raw.  I'll fix.

File: mock/el7-scl.cfg (hunk 2 above)
Comment (ianballou) [reply mock/el7-scl.cfg]: Paste mistake!

**Comment by ianballou** [comment]
Hitting the same build issue that #4741 is having.

File: mock/el7-scl.cfg [hunk 3]
```diff
@@ -1,7 +1,7 @@
 config_opts['root'] = 'foreman-scl-7-x86_64'
 config_opts['target_arch'] = 'x86_64'
 config_opts['legal_host_arches'] = ('x86_64',)
-config_opts['chroot_setup_cmd'] = 'install @buildsys-build scl-utils-build tfm-build yum vim'
+config_opts['chroot_setup_cmd'] = 'install @buildsys-build scl-utils-build tfm-build yum vim rh-postgresql12-scldevel'
 config_opts['dist'] = 'el7'  # only useful for --resultdir variable subst
 # beware RHEL use 7Server or 7Client
```
Comment (jlsherrill) [inline mock/el7-scl.cfg]: we probably don't want to update this file

File: packages/pulpcore/postgresql-evr/rh-postgresql12-postgresql-evr-0.0.1.tar.gz [hunk 4]
```diff
@@ -0,0 +1 @@
+../../../.git/annex/objects/97/pG/SHA256E-s13799--70a4b233190758af56b82f109d87f132d89a74180d53f2318bb41cae27a09b51.tar.gz/SHA256E-s13799--70a4b233190758af56b82f109d87f132d89a74180d53f2318bb41cae27a09b51.tar.gz
```
Comment (jlsherrill) [inline packages/pulpcore/postgresql-evr/rh-postgresql12-postgresql-evr-0.0.1.tar.gz]: shouldn't this be named just 'postgresql-evr-0.0.01.tar.gz'  ?

File: mock/el7-scl.cfg (hunk 3 above)
Comment (ianballou) [reply mock/el7-scl.cfg]: oh that was just for local testing, my bad.

File: comps/comps-katello-pulpcore-el7.xml [hunk 5]
```diff
@@ -97,7 +97,8 @@
       <packagereq type="default">python3-urllib3</packagereq>
       <packagereq type="default">python3-whitenoise</packagereq>
       <packagereq type="default">python3-yarl</packagereq>
-      <packagereq type="default">python3-zipp</packagereq>
+	  <packagereq type="default">python3-zipp</packagereq>
+	  <packagereq type="default">rh-postgresql12-postgresql-evr</packagereq>
```
Comment (jlsherrill) [inline comps/comps-katello-pulpcore-el7.xml]: indentation issue here

**Comment by jlsherrill** [comment]
@ehelms suggested we not worry about el8 builds at the moment while we wait for the build team to continue their work.

@ehelms @ekohl @evgeni   any other comments before merging?

**Pull request merged**